content/Projects/Observability/RHOBS/mst.md

```diff
@@ -0,0 +1,25 @@
+# RHOBS: MST
+
+> For RHOBS Overview see [this document](README.md)
+
+TBD
+
+## Support
+
+TBD
+
+## Service Level Agreement
+
+![SLO](../../../assets/slo-def.png)
+
+If you manage [“Observatorium”](../observatorium.md) the Service Level Objectives can go ultra high in all dimensions like availability, and data loss. The freshness aspect for read APIs is trickier as among other things it also depends on client collection pipeline availability which is out of Observatorium scope.
+
+RHOBS as Observatorium as a Service has currently established following, default, Service Level Objectives. This is based on infrastructure dependencies we have listed [here (intenral)](https://visual-app-interface.devshift.net/services#/services/rhobs/app.yml).
```

Comment:
```suggestion
RHOBS as Observatorium as a Service has currently established the following, default, Service Level Objectives. This is based on the infrastructure dependencies we have listed [here (internal)](https://visual-app-interface.devshift.net/services#/services/rhobs/app.yml).
```

Reply:
What do we mean by "RHOBS as Observatorium as a service"?

Reply:
Also drop the commas around "default."